### DISCUSSION TITLE
Student Developer pack location issue

### DISCUSSION THREAD

**Discussion by VartikaShekhar** [community/community#126601] [GitHub Education]
### Hi everyone,

Hi everyone,

I study in University of Wisconsin-Madison. But I am in another country for the summer and want to use the student developer pack right now but, my verification will not go through since it says it doesn't find my registered uni in my location (which is true). So how to I go about this, it would really help me to use the services in the package right now?

**Comment by aliffian425** [comment]
you can try to explore alternative options, depending on your specific needs, you could also explore alternative options for developer tools and resources that are not location-dependent or have less stringent verification processes.

It's important to note that the GitHub Student Developer Pack is a privilege offered to currently enrolled students, and the verification process is in place to maintain the integrity of the program. Therefore, it's crucial to follow the guidelines and policies set by GitHub.

**Reply by VartikaShekhar** [reply]
Is there no way for me to get access to the student pack as I am enrolled with the university and will go back after the summer?

**Reply by aliffian425** [reply]
maybe wait until you return, i know, while this may not be the ideal solution, it ensures that you follow the program's guidelines.

**Comment by davevad93** [comment]
Hi @VartikaShekhar ,

Check this old discussion #51930 about a problem similar to yours and read the comment of one of the Community Mantainers, it might be helpful.

**Reply by VartikaShekhar** [reply]
Thanks! But there is no documentation that I could use to show that I am learning remotely (as I am not). I am just on vacation and would return only after summer ends.

**Reply by davevad93** [reply]
Hey @VartikaShekhar , if you're going back home at the end of summer I would advise you to wait until then, you'll not lose the right to claim your student benefits.
